rotkehlchen/api/rest.py

```diff
@@ -4444,9 +4445,23 @@ def get_config_arguments(self) -> Response:
                 'value': self.rotkehlchen.args.sqlite_instructions,
                 'is_default': self.rotkehlchen.args.sqlite_instructions == DEFAULT_SQL_VM_INSTRUCTIONS_CB,  # noqa: E501
             },
+            'loglevel': {
+                'value': (current_level := logging.getLevelName(logging.getLogger().level)),
+                'is_default': current_level == DEFAULT_LOGLEVEL,
+            },
         }
         return api_response(_wrap_in_ok_result(config), status_code=HTTPStatus.OK)
 
+    def update_log_level(self, loglevel: str) -> Response:
+        """Update the current log level"""
+        numeric_level = getattr(logging, loglevel)
+        logging.getLogger().setLevel(numeric_level)
+
```

Comment:
is this `logging.getLogger()` the same as the `logger` variable that we have at the start of the file?